test/system/inspect.spec.ts

```diff
@@ -48,7 +47,7 @@ describe('inspect', () => {
         .connectDep('jinja2', 'markupsafe')
         .build();
 
-      expect(result.dependencyGraph).toEqualDepGraph(expected);
+      expect(result.dependencyGraph.equals(expected)).toBeTruthy();
```

Comment:
After bumping node version and jest version there's TS complaint about not finding this method, instead of fixing the complaint think it's easier and cleaner to just use the `equals` method directly offered by the `dep-graph` lib. Here both `dependencyGraph` and `expected` are of types `DepGraph` so they could readily be compared this way. And the `equals` method does deep equal according to the library :) see [here](https://github.com/snyk/dep-graph/blob/a27eded6b412cecb7ce04cb524dbeca517fc8c87/src/core/dep-graph.ts#L188). (Actually this is also how we compare graph in the new Yarn parser, which I like :D)